src/AbstractEnum.php

```diff
@@ -15,11 +17,13 @@
      * them in a local property for performance, before returning them.
      *
      * @param callable|null $keysCallback
+     * @param bool          $classPrefixed      True if you want the enum class prefix on each keys, false otherwise.
+     * @param string        $namespaceSeparator Only relevant if $classPrefixed is set to true.
      *
      * @return array a hash with your constants and their value. Useful for
      *               building a choice widget
      */
-    final public static function getConstants($keysCallback = null)
+    final public static function getConstants($keysCallback = null, $classPrefixed = false, $namespaceSeparator = '_')
```

Comment:
Maybe `$classPrefixed` can be `true` by default?

Most of the key, translator will be used for a proper format. Using class prefix make more sense IMHO.